include/caf/caf_overview.rst

```diff
@@ -0,0 +1,182 @@
+.. _caf_overview:
+
+Common Application Framework overview
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Common Application Framework (CAF) is a set of ready-for-use modules and events built on top of :ref:`event_manager`.
+Using CAF allows you to have a consistent event-based architecture in your application.
+
+One of the applications that are using the CAF modules in |NCS| is :ref:`nrf_desktop`.
+
+In an event-based application, parts of the application functionality are separated into isolated modules that communicate with each other using Event Manager's events.
+These events are defined either by CAF or by the application.
+They can be submitted by modules and other modules can subscribe and react to them.
+
+.. figure:: ../../doc/nrf/images/caf_overview.svg
+   :alt: Common Application Framework architecture overview
+
+   Common Application Framework architecture overview
+
+Every CAF module can use additional libraries or drivers.
+The drivers can be used to communicate with hardware.
+The module can then broadcast information from the driver using Event Manager's events (for example, submit a :c:struct:`button_event` on a button press).
+It can also forward information from application to the driver (for example, display a given LED effect upon receiving :c:struct:`led_event`).
+
+In a user application, you can use any number of CAF modules with your own custom modules, and they can use both CAF and custom events.
```

Comment:
In your?
Sometimes there is  "your/ in your" and then "in a user". It requires alignment.